sim/sim_questa.tcl

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 CSEM
```

Comment:
If this is not an issue for CSEM, I would prefer if you use version 2.1 of the Solderpad License (that is the version currently used by Vicuna) for new files to avoid having files licensed under different versions.

Also, does CSEM require that parts of the license text are included at the top of every file or would a short one-liner referencing the [license file](https://github.com/vproc/vicuna/blob/main/LICENSE.txt), as used in the other source files, be sufficient? See for instance the header of `sim_vivado.tcl`:

https://github.com/vproc/vicuna/blob/51bd47f9f1f746e6b4a924b5c0805e99d20a76e5/sim/sim_vivado.tcl#L2

Finally, if you could add an SPDX license identifier as well, that allows software to easily identify the license that applies to a file. The recommended SPDX identifier for version 2.1 of the Solderpad license is `SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1`.

Reply:
I have a meeting in a few minutes, I will discuss this there.